lib/settings_base.py

```diff
@@ -600,6 +600,7 @@ def JINJA_CONFIG():
             'js/zamboni/buttons.js',
             'js/zamboni/tabs.js',
             'js/common/keys.js',
+            'js/common-min.js',
```

Comment:
Ah, so I would assume that common-all would be the file that it is generated from? It seems as though replacing common-min with common-all still fixes the issue.